src/main/java/org/embulk/input/jira/JiraInputPlugin.java

```diff
@@ -173,7 +173,7 @@ public ConfigDiff guess(ConfigSource config)
     private ConfigSource createGuessConfig()
     {
         ConfigSource configSource = Exec.newConfigSource();
-        configSource.set("guess_plugins", new ObjectMapper().createArrayNode().add("jsonpath"));
+        configSource.set("guess_plugins", new ObjectMapper().createArrayNode().add("jira"));
```

Comment:
```suggestion
        return Exec.newConfigSource()
        .set("guess_plugins", ImmutableList.of("jira"))
        .set("guess_sample_buffer_bytes", GUESS_BUFFER_SIZE);
```
It's easier to read with chaining.

Reply:
I think these should be better addressed on #53, the scope of this PR is about removing `embulk-parser-jsonpath` along with adapting the unit tests.
@tvhung83 how about I clone your comments to #53, proceed to merge this PR (if there aren't any further issue belong to the scope above), then we pick up from there?

Reply:
Ok, please go ahead.

Reply:
To be addressed on https://github.com/treasure-data/embulk-input-jira/pull/53/files#r257136299